frontend/src/components/DetailsDialog.svelte

```diff
@@ -28,37 +29,41 @@
             "blob",
             job.version,
             `${stem}${ext}#L${lineNum}C${colNum}-L${lineNum}`,
-          ].join("/")
+          ].join("/"),
         );
         link.rel = "noreferrer";
         link.target = "_blank";
         link.textContent = orig;
         return link.outerHTML;
-      }
+      },
     );
   }
 </script>
 
 <Dialog bind:open bind:fullscreen={isErrorDialog} sheet>
   {#if selectedJob}
     <Header>
-      <Title>Job #{selectedJob.id}</Title>
+      <Title id="details-title">
+        <div id="details-title-left">Job #{selectedJob.id}</div>
+        <div id="details-title-right">
+          <div>{repoToString(selectedJob.repository)}</div>
+          <div>{selectedJob.job_type.display_name}</div>
+        </div>
+      </Title>
     </Header>
     <Content>
       {#if selectedJob.status.name === "completed"}
-        {#each selectedJob.artifact_urls as artifact}
-          <a href={artifact.url} target="_blank" rel="noreferrer"
-            >{artifact.slug}</a
-          >
-          <br />
-        {/each}
+        <BuildArtifacts {selectedJob}></BuildArtifacts>
+        {#if selectedJob.job_type.name === "git-web-hosting-preview"}
```

Comment:
(note to self) This seems to be the place in the UI that decides whether a book can be approved... but it seems like the BE allows other options? (maybe allow other options here or remove other options there?)

Reply:
This seems to be the place in the UI that decides whether a ~book can be approved~ job can be used to get a list of books to approve.